src/js/constants/accessKey.js

```diff
@@ -0,0 +1,8 @@
+import * as joobyCodec from 'jooby-codec';
+
+
+export const LENGTH_BYTES = 16;
+export const DEFAULT = joobyCodec.utils.getHexFromBytes(
```

Comment:
DEFAULT_BYTES and DEFAULT_HEX ?

Reply:
Renamed DEFAULT to DEFAULT_HEX, DEFAULT_BYTES will not be used